modules/olm-installing-operators-from-operatorhub.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id="olm-installing-operators-from-operatorhub_{context}"]
+= Installing Operators from the OperatorHub
```

Comment:
This is just pulling this content out from the `olm-adding-operators-to-cluster.adoc` assembly and making it its own module.